### PR TITLE
add voyageai example

### PR DESCRIPTION
**Pull request by yiweny**

### CODE REVIEW

File: examples/llm_embedding.py [hunk 1]
```diff
@@ -81,15 +85,38 @@ def __call__(self, sentences: List[str]) -> Tensor:
...
+
+    def __init__(self, model: str = 'voyage-01'):
+        # Please run `pip install voyageai` to install the package
+        self.model = model
+
+    def __call__(self, sentences: List[str]) -> Tensor:
+        import voyageai
+        voyageai.api_key = api_key
+        from voyageai import get_embeddings
+        items = get_embeddings(sentences, model=self.model)
```
Comment (weihua916) [inline examples/llm_embedding.py:100]: Just curious, what's the type of `items`? Can we have the same type annotation for the other API embeddings as well?

Comment (yiweny) [reply examples/llm_embedding.py:100]: `List[List[Float]]`

**Review by weihua916 — APPROVED** [review]
Thanks! How does it perform?

**Comment by yiweny** [comment]
> Thanks! How does it perform?

still running experiments.

**Pull request merged**